### DISCUSSION TITLE
JPEG2000 tile info

### DISCUSSION THREAD

**Discussion by scossu** [libvips/libvips#3576] [Q&A] [answered]
I am trying to correct a large batch of JPEG2000 images that were created with odd tile sizes and whose originals are not available.

I see that `vips_jp2ksave ()` has parameters for tile sizes, which I can use to remediate the corrupted images. But I am also looking through the libvips documentation for a command to display tile info, so that I can run it on an image collection and find the ones that need remediation without having to rewrite all of them. 

Nothing came up as an obvious solution. Is there a command I can use?

Thanks.

**Comment by jcupitt** [comment, marked as answer]
Hi @scossu, no, libvips does not expose tile size info for jp2k. 

OpenJPEG have a tool called `opj_dump`, I think that would work for you.
